src/components/Analytics/analytics.test.js

```diff
@@ -1,84 +0,0 @@
-import { stub } from "sinon";
```

Comment:
should remove @next/ga and everythng related like ga config

Reply:
> Aesthetic comments
> 
> 1.`EXAMPLE.OPENATTESTATION.COM` is going on a new line ONLY when details are open on smaller screen
> ![image](https://user-images.githubusercontent.com/503185/66182648-89110900-e6a8-11e9-9f73-38419bd5bfbe.png)
> 
> ===============================================================================
> 
> 1. Details are sometimes over the document, sometimes not
>    ![image](https://user-images.githubusercontent.com/503185/66182863-3257ff00-e6a9-11e9-847b-6c5b7053322c.png)
>    ![image](https://user-images.githubusercontent.com/503185/66182874-3dab2a80-e6a9-11e9-819c-1c5554df2544.png)

can create a new issue for this